vendor/bat-native-ads/src/bat/ads/internal/ml/data/data.cc

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <limits>
+#include <map>
+#include <stdexcept>
+
+#include "bat/ads/internal/ml/data/data.h"
+
+namespace ads {
+namespace ml {
+namespace data {
```

Comment:
`data` namespace not needed as the class is named Data

Reply:
Does it work like this if the base class is named `Data`, but the concrete classes in the same namespace are `TextData` and `VectorData`?

Reply:
correct so as an example `ads::ml::Data`, `ads::ml::TextData`

Reply:
Sorry, wrong place for the comment :(